NuKeeper.Tests/RepositoryInspection/ProjectFileReaderTests.cs

```diff
@@ -136,7 +142,7 @@ public void ResultIsReiterable()
 
             foreach (var package in packages)
             {
-                Assert.That(package, Is.Not.Null);
+                PackageAssert.IsPopulated(package);
             }
 
             Assert.That(packages.Select(p=>p.Path), Is.All.EqualTo(path));
```

Comment:
While I agree that this test will test whether or not the result is iterable, it has extra detail in the tests that could confuse the intent for somebody reading the test